app/views/profile/index.html.slim

```diff
@@ -17,6 +17,8 @@
     class: 'h5 blue sans-serif underline hint--top hint--no-animate',
     tabindex: 0,
     'aria-label': t('tooltips.profile_idv'))
+  - unless current_user.decorate.identity_not_verified?
```

Comment:
maybe we can actually return the correct partial from the decorator to avoid more conditionals in our views? so `current_user.account_verification_header` -- this would return `app/views/shared/_null.html.slim` for users who should see nothing.